src/blockstore.js

```diff
@@ -153,7 +153,7 @@ function createBaseStore (store) {
      *
      * @returns {Promise<void>}
      */
-    async close () {
+    close () {
```

Comment:
this should still be async, in the original PR alan mentions a scenario where not labeling async causes a user using traditional then catch to then get an uncaught exception error. Perhaps because the store , I'm guessing this is dependent on the underlying blockstore implementation. Aside from this it's probably still a good idea to label it as async/ which mirrors the rest of the new interfaces